ci-operator/step-registry/openshift/upgrade/qe/test/canary/openshift-upgrade-qe-test-canary-chain.yaml

```diff
@@ -0,0 +1,24 @@
+chain:
+  as: openshift-upgrade-qe-test-canary
+  steps:
+  - ref: idp-htpasswd
+  - ref: cucushift-pre
+  - ref: openshift-extended-upgrade-pre
+  - ref: cucushift-upgrade-prepare
+  - ref: cucushift-upgrade-prehealthcheck
+  - ref: mco-conf-day2-create-custom-mcp
+  - ref: cucushift-upgrade-eus-pause-worker-mcp
+  - ref: cucushift-ota-preupgrade
+  - ref: cucushift-upgrade-toimage
+  - ref: cucushift-canupgrade-unpause-worker
```

Comment:
We also have an existing step - `cucushift-upgrade-eus-unpause-worker-mcp` can be re-used here, similar to `cucushift-upgrade-eus-pause-worker-mcp`

Reply:
`cucushift-canupgrade-unpause-worker` is much different from `cucushift-upgrade-eus-pause-worker-mcp`

Reply:
The difference that you mean is "removing label and remove custom mcp"? We have `mco-conf-day2-create-custom-mcp` and `cucushift-upgrade-eus-pause-worker-mcp`, accordingly I am expecting some steps like mco-conf-day2-destory-custom-mcp` and `cucushift-upgrade-eus-unpause-worker-mcp` steps.

Reply:
Let me take a look